docs/source/app-configuration.rst

```diff
@@ -0,0 +1,160 @@
+App Integration Configuration
+=============================
+
+Overview
+--------
+
+To further StreamAlert's abilities past ingesting logs from services that offer support for sending to AWS Lambda,
+the package includes a method to fetch logs from RESTful APIs to be forwarded to StreamAlert. This capability is added through the
+use of StreamAlert Apps.
+
+StreamAlert Apps are designed to run on scheduled intervals and collect logs from a third-party API to be relayed on to the Rule Processor
+for processing.
+
+
+Concepts
+--------
+
+StreamAlert Apps are made possible through the use of AWS technologies:
+
+* `AWS EC2 Simple System Manager's Parameter Store <https://aws.amazon.com/ec2/systems-manager/parameter-store/>`_
+* `AWS Lambda Invocations via Scheduled Events <http://docs.aws.amazon.com/lambda/latest/dg/with-scheduled-events.html>`_
+
+
+Supported Services
```

Comment:
mention there's more to come (list onelogin, gsuite and their associated github issue #'s)